docs/mkdocs-base.yml

```diff
@@ -46,7 +46,10 @@ nav:
                   - Guide:
                           - user/guide/index.md
                   - Manual :
+                          - Introduction: user/manual/index.md
```

Comment:
This line is no longer needed as line 50 supersedes it

Reply:
Should end up with something like

![image](https://github.com/kartoza/sawps/assets/100694811/67be685e-7455-40f6-96cd-96cba66b300c)

Reply:
OK cool let me updaate